screeps-game-api/src/constants.rs

```diff
@@ -979,3 +995,60 @@ impl TryFrom<Value> for ResourceType {
         })
     }
 }
+
+// TODO: reactions
+// TODO: boosts
+
+pub const PORTAL_UNSTABLE: u32 = 10 * 24 * 3600 * 1000;
+pub const PORTAL_MIN_TIMEOUT: u32 = 12 * 24 * 3600 * 1000;
+pub const PORTAL_MAX_TIMEOUT: u32 = 22 * 24 * 3600 * 1000;
+
+pub const POWER_BANK_RESPAWN_TIME: u32 = 50000;
+
+pub const INVADERS_ENERGY_GOAL: u32 = 100000;
+
+pub const SYSTEM_USERNAME: &str = "Screeps";
+
+pub const SIGN_PLANNED_AREA: &str =
+    "A new Novice or Respawn Area is being planned somewhere \
+     in this sector. Please make sure all important rooms are reserved.";
+
+pub const POWER_LEVEL_MULTIPLY: u32 = 1000;
+pub const POWER_LEVEL_POW: u32 = 2;
+pub const POWER_CREEP_SPAWN_COOLDOWN: u32 = 8 * 3600 * 1000;
+pub const POWER_CREEP_DELETE_COOLDOWN: u32 = 24 * 3600 * 1000;
+pub const POWER_CREEP_MAX_LEVEL: u32 = 25;
+pub const POWER_CREEP_LIFE_TIME: u32 = 5000;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+#[cfg_attr(feature = "constants-serde", derive(Serialize, Deserialize))]
+pub enum PowerClass {
+    #[cfg_attr(feature = "constants-serde", serde(rename = "operator"))]
```

Comment:
You just removed those higher up. Can this one also be changed to `#[serde(rename(...)]`?

Reply:
Hm - I mean we could! I removed the ones above because we used them within the codebase, while these are still feature gated so we don't generate extra code. It is somewhat arbitrary that serde implementations are feature gated this way but nothing else is, though.

Maybe we should remove these gates and just have everything implement it all the time!

Reply:
Do you remember why we wanted to feature gate them? While we might save some space, this is something that would trip newcomers.

Reply:
I don't really - I think I just personally needed them and didn't want to bloat the binary?

I'll remove the feature gate - they're useful enough.